docs/static/playground/pxt-common/pxt-core.d.js

```diff
@@ -526,3 +526,11 @@ declare namespace Math {
     //% shim=Math_::idiv
     function idiv(x: number, y: number): number;
 }
+
+declare namespace _lifecycle {
+    //% shim=__lifecycle::onCodeStart
+    export function onCodeStart(arg: any): void;
```

Comment:
we should move this namespace those under "control" where most of the runtime function exist.